eng/common/cross/build-rootfs.sh

```diff
@@ -310,6 +312,8 @@ done
 
 if [[ "$__BuildArch" == "armel" ]]; then
     __LLDB_Package="lldb-3.5-dev"
+elif [[ "$__BuildArch" == "arm" && "$__CodeName" == "alpine" ]]; then
```

Comment:
It is not obvious that this applies to alpine 3.13 only.

Should we delete support for the unversioned `alpine` from the script and support explicit versions only?

Reply:
Ah, missed the version check.
```suggestion
elif [[ "$__BuildArch" == "arm" && "$__CodeName" == "alpine" && "$__AlpineVersion" == "3.13" ]]; then
```